src/common/components/clusterConfiguration/CertificateFields.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { useField, useFormikContext } from 'formik';
+import { Grid } from '@patternfly/react-core';
+import { CheckboxField, HelperText, PopoverIcon, UploadField } from '../ui';
+import { TrustedCertificateFieldsType } from '../../types';
+import { useTranslation } from '../../hooks/use-translation-wrapper';
+
+import './ProxyFields.css';
```

Comment:
Why this CSS? It has a class name for proxy, so I don't think it applies to this component.

Reply:
True, copy & paste problem